src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -82,6 +106,19 @@ def update_data(self):
             subid = (if_index,) + iptuple
             self.arp_dest_map[subid] = machex
             self.arp_dest_list.append(subid)
+
+    def update_data(self):
+        self.arp_dest_map = {}
+        self.arp_dest_list = []
+        """
+        update arp table of host.
+        In case of multi-asic platform, get host arp table
+        from kernel and namespace arp table from NEIGH_TABLE in APP_DB
+        in each namespace.
+        """
+        self._update_host_data()
```

Comment:
>_update_host_data [](start = 13, length = 17)

For single ASIC, we should treat 'NEIGH_TABLE' as the ground truth. No need to check arp_table.
And we need to adjust test for single ASIC.

Reply:
Updated to read 'NEIGH_TABLE' for single asic as well. Updated mock db to reflect this change.